.gitignore

```diff
@@ -127,6 +127,8 @@ test_udp
 test_large_msg
 test_use_fd_ipc
 test_use_fd_tcp
+test_pre_allocated_fd_ipc
+test_pre_allocated_fd_tcp
```

Comment:
Ah, ja... could you send a PR?

On Thu, Feb 11, 2016 at 3:21 PM, Luca Boccassi <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/zeromq/libzmq/pull/1790#discussion_r52606302>:
>
> > @@ -127,6 +127,8 @@ test_udp
> >  test_large_msg
> >  test_use_fd_ipc
> >  test_use_fd_tcp
> > +test_pre_allocated_fd_ipc
> > +test_pre_allocated_fd_tcp
>
> These 2 don't exist anymore, they were renamed to test_use_fd_tcp :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zeromq/libzmq/pull/1790/files#r52606302>.
>